packages/two_dimensional_scrollables/lib/src/table_view/table.dart

```diff
@@ -829,17 +984,35 @@ class RenderTableViewport extends RenderTwoDimensionalViewport {
       _paintCells(
         context: context,
         offset: offset,
-        leading: const TableVicinity(column: 0, row: 0),
-        trailing:
+        leadingVicinity: TableVicinity.zero,
+        trailingVicinity:
             TableVicinity(column: _lastPinnedColumn!, row: _lastPinnedRow!),
       );
     }
   }
 
+  @override
+  RenderBox? getChildFor(
+    ChildVicinity vicinity, {
+    bool allowMerged = true,
+  }) {
+    return super.getChildFor(vicinity) ??
```

Comment:
Did you mean to change this to the suggestion in that other comment?

Reply:
I hadn't because I want to be able to assert that if we requested a child that returned null (using the super getChildFor first), then it is for certain covered by a merged cell.

Reply:
Or, maybe stated better, I want to see if there is a canonical child first, and if not, need to ensure we have not lost a child so it asserts the vicinity returning null is in fact covered by a merged cell.